### PR TITLE
Adding Alignment to project goal(s) category

### PR DESCRIPTION
**Pull request by yanfisher**
Other small edits to Vision group

Signed-off-by: Yan Fisher <yfisher@redhat.com>

### CODE REVIEW

**Review by mestery — APPROVED** [review]
I think the approach of putting in a placeholder for each subgroup leader to update makes some sense, because we are going to force the subgroup folks to make a GitHub commit! :)

**Comment by pdp2shirts** [comment]
@yanfisher if you merge this I will go in and create a PR for the sections I need to fill out.

**Comment by mestery** [comment]
@pdp2shirts Yan can't merge this as he's not a maintainer. I'll merge it now and then you can submit a followup.

**Pull request merged**
